src/main/resources/db/mysql/petclinic_db_setup_mysql.txt

```diff
@@ -22,7 +22,14 @@
    scripts (or set "spring.datasource.initialization-mode=always" the first time you run the app).
 
 3) Run the app with `spring.profiles.active=mysql` (e.g. as a System property via the command
-   line, but any way that sets that property in a Spring Boot app should work).
+   line, but any way that sets that property in a Spring Boot app should work). For newer versions
+   of Spring Boot, those will fork the JVM that executes the webserver, which means properties on
+   the command line are lost. Use
+   
+   mvn spring-boot:run\
+   -Dspring-boot.run.jvmArguments="-Dspring.profiles.active=mysql"
+
+   To make mysql work with forking.
```

Comment:
Not sure why this sentence was added or why you should mention forking. That property works with or without forking.

Reply:
Would it be better to just have (3) say:
```
3) Run the app with `spring.profiles.active=mysql` (e.g. as a System property via the command
   line, but any way that sets that property in a Spring Boot app should work, like
   -Dspring-boot.run.profiles=mysql)
```

Reply:
I don't think so as it's maven specific and there's no connection to it the way you're suggesting

Reply:
I'm not sure what would be better to write.

I thought of amending the docs because I recently wanted to run this using the provided
`./mvnw spring-boot:run`, then wanted to switch to mysql, and tried  `./mvnw -Dspring.profiles.active=mysql spring-boot:run` and `./mvnw spring-boot:run -Dspring.profiles.active=mysql`, and neither worked. I thought if for a beginner, the documentation showed a way to fix that issue, it could be helpful.

Reply:
Absolutely and your proposal is very much appreciated. Let me take over and I'll polish it if that's ok with you?

Reply:
sure, thanks!
My apologies I'm not that familiar (yet) with your way of doing things.

Reply:
No need to apologize for anything. This back and forth is to be expected and not indicative of a problem. Thanks again for your contribution.